### ISSUE TITLE
Advanced Inscriber Auto Ejecting without an importer/exporter attached

### ISSUE THREAD

**Issue by jkwaak1980** [Technici4n/AE2Things-Forge#7]
I am playing through ATM7 and I think I may have found a bug.  I noticed that the advanced inscriber was auto ejecting the results of the crafting process to my AE2 network even though I did not have an importer hooked up to it.  Is this an intended action or something else.

**Comment by Technici4n** [comment]
Yep, that's intended! :)

**Comment by necauqua** [comment]
Um, this could've maybe be made toggleable with a button in the UI or something?
That's like, totally unexpected behavior that is never explained anywhere, at least add that to the tooltip.
I've spent a lot of time trying to figure out why the hell do circuit components end up in the system.
I get that this is probably good for the "pure AE" circuit setup, but for us peasants using laserio instead this is confusing.

Now that I found this issue and understood that this is indeed indended behavior that I cannot change, I had to fix my spaghetti (by making even worse spaghetti) to not pass a bunch of channels through inscribers and just power them with quartz fiber :)
